networkx/algorithms/dag.py

```diff
@@ -489,6 +489,8 @@ def dag_longest_path(G, weight='weight', default_weight=1):
     --------
     dag_longest_path_length
     """
+    if not G:
```

Comment:
It would be great if you added a check to make sure that this works as intended (and that way if we accidentally change it later, we will know).

Reply:
No problem, done.